docs/testing/child.md

```diff
@@ -0,0 +1,10 @@
+This page contains some variables that are being imported in other files.
+There should be only VARIABLES START and VARIABLES END in red:
+
+<div style="color:red">
+VARIABLES START
+<variable name="title">Title</variable>
+<variable name="child_src">testimportvariables.md</variable>
+<variable name="child_src_working">testimportvariables</variable>
+VARIABLES END
+</div>
```

Comment:
This file needs to be removed as well?

Other than this I have no other comments, I assume that the design is mostly fixed and this PR is no longer WIP.